matlab/MeCorr/MATLAB_CODE_allrxns.m

```diff
@@ -0,0 +1,419 @@
+      %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+      % this code reproduces all the main text figures from the manuscript (Shen et al)
+      %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Bulk methylation model:
+% 1) cd ./../scripts. Run make_eGEM to create bulk methylation model
+% 2) cd ./../MeCorr. Run section 1 and 3 of this script. Only run Section 2 if you want to 
+% run the long for-loop
+% 3) Parameters of interest to vary: epsilon_methylation, model2
+cd ./../scripts
+make_eGEM
+cd ./../MeCorr
+%clearvars -except min_model % If don't want to run make_eGEM
+model2 = min_model;
+epsilon_methylation = 1E-1;
+%rxnpos  = find(ismember(min_model.rxns,'LYSMTF1n'));
+minfluxflag = 1; % 0: no Pfba, 1: Pfba 
+% 4) Change name of files to which variables are saved. End of section 1
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+       % impact of basal metabolic state of CCLE cell lines on sensitivity to demethylase inhibitors
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+load supplementary_software_code  ctd2celllineidname exptidcelllinemediamatch ctd2celllineidname_id r* 
+%data from seashore-ludlow study, contains cell line names, growth media
+load supplementary_software_code  ctd2compoundidname_id drug_auc_expt ctd2compoundidname_name 
+%data from seashore-ludlow study, contains drug names, drug sensitivity data
+load supplementary_software_code celllinenames_ccle1 ccleids_met ccle_expression_metz  
+%contains CCLE cell line names, gene expression data (z-transformed)
+load supplementary_software_code hcommon_exptdat hcommon1 
+%data from seashore-ludlow study, contains drug names, drug sensitivity data for cell lines that were screened against all 8 hme inhibitors
+load supplementary_software_code hdacexpfcs hdacexpallgeneids 
+%contains gene expression data after treatment with hdac inhibitors
+
+% Converting arrays to tables and renaming some Variables to make

```

Comment:
Once you prettify these variables, you should save them in `vars` and just load them rather than formatting them during every iteration within this script.